### PR TITLE
Update the available ruby version to 2.3.0 or later

### PR DESCRIPTION
**Pull request by kachick**

### CODE REVIEW

File: sbpayment.gemspec [hunk 1]
```diff
@@ -17,6 +17,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.required_ruby_version = '>= 2.3'
```
Comment (kachick) [inline sbpayment.gemspec]: Talked with @banyan. Down to `2.2`. And add CI tests with 2.2 too.

Comment (kachick) [inline sbpayment.gemspec]: @banyan Do you know another smart way of http://qiita.com/sawanoboly/items/bd7c12258fe9f29646d5? :sweat_smile:

Comment (banyan) [inline sbpayment.gemspec]: I don't know. or this one? http://www.jetthoughts.com/blog/tricks/2014/11/24/run-tests-against-multiple-ruby-versions-on-circleci.html

Comment (yuya-takeyama) [inline sbpayment.gemspec]: @kachick @banyan 
How about just using Travis CI?

Comment (kachick) [inline sbpayment.gemspec]: Sounds good to me. @banyan How do you think?

Comment (kachick) [inline sbpayment.gemspec]: This time I have used the https://github.com/quipper/sbpayment.rb/pull/33#discussion_r51972761 way in https://github.com/quipper/sbpayment.rb/commit/269f0d9a9d33eb30a9a2908d010d20d2680137e3.

**Comment by banyan** [comment]
sorry, could you change the commit message since it's contradicting. 37d6d6b

**Comment by banyan** [comment]
ah, never mind.

**Pull request merged**